portal/src/app/components/home/components/experiment/components/table-options/TableOptions.module.scss

```diff
@@ -0,0 +1,41 @@
+@import "src/styles/variables-keys";
+@import "src/styles/z-index";
```

Comment:
you can remove this import, not in use

Reply:
This one is used: `@import "src/styles/variables-keys";`
Removing `@import "src/styles/z-index";`